pandas/compat/pyarrow.py

```diff
@@ -4,6 +4,8 @@
 
 from pandas.util.version import Version
 
+PYARROW_INSTALLED = None
```

Comment:
is this necessary?

Reply:
Probably not (I was first doing it with a function and a global variable, but that complexity wasn't actually needed)